### PR TITLE
Implement Introspection using Manual Instrumentation Events.

### PR DESCRIPTION
**Pull request by florian-kuebler**
This removes introspection from all events and all "client" specific
code for it, and instead use the new manual instrumentation events.

Collecting the events is still handled differently
(using TracingListeners instead of the injection of liborbit.so),
but all protos used, as well as all code that handles the events
is the same as for manual instrumentation.

This allows us to simplify a lot of code and remove all the special
handling.

Bug: http://b/196032048 http://b/182848566
Test: Take a capture on OrbitTest and --devmode, start introspection

### CODE REVIEW

File: src/OrbitGl/IntrospectionWindow.cpp [hunk 1]
```diff
@@ -13,7 +13,187 @@ using orbit_client_data::CaptureData;
...
-IntrospectionWindow::IntrospectionWindow(OrbitApp* app) : CaptureWindow(app) {
+namespace {
+inline void HandleCaptureEvent(const orbit_api::ApiScopeStart& scope_start,
+                               orbit_capture_client::ApiEventProcessor* api_event_processor) {
+  orbit_grpc_protos::ApiScopeStart api_event;
+  scope_start.CopyToGrpcProto(&api_event);
+  api_event_processor->ProcessApiScopeStart(api_event);
+}
+
+inline void HandleCaptureEvent(const orbit_api::ApiScopeStop& scope_stop,
```
Comment (dimitry-) [inline src/OrbitGl/IntrospectionWindow.cpp:24]: does inline actually give us anything here? I suspect it is inlined anyways and we should probably leave it to compiler to decide. Unless this is absolutely necessary.

File: src/Service/TracingHandler.cpp [hunk 2]
```diff
@@ -40,23 +39,87 @@ void TracingHandler::Start(CaptureOptions capture_options) {
   }
 }
 
+namespace {
+inline void CreateCaptureEvent(const orbit_api::ApiScopeStart& scope_start,
```
Comment (dimitry-) [inline src/Service/TracingHandler.cpp:43]: same for inline here..

**Review by dimitry- — APPROVED** [review]
otherwise lgtm

File: src/ClientProtos/capture_data.proto [hunk 3]
```diff
@@ -146,7 +146,7 @@ message TimerInfo {
   enum Type {
     kNone = 0;
     kCoreActivity = 1;
-    kIntrospection = 2;
+    // kIntrospection = 2;
```
[the remark this answers is not in the document]
Comment (florian-kuebler) [reply src/ClientProtos/capture_data.proto:149]: Unfortunately, they aren't

File: src/OrbitGl/IntrospectionWindow.cpp (hunk 1 above)
Comment (florian-kuebler) [reply src/OrbitGl/IntrospectionWindow.cpp]: Removed it.

File: src/Service/TracingHandler.cpp (hunk 2 above)
Comment (florian-kuebler) [reply src/Service/TracingHandler.cpp]: Removed it.

**Pull request merged**